### ISSUE TITLE
Ditching Webpack

### ISSUE THREAD

**Issue by johnthepink** [NewSpring/newspring-app#189]
I've been working on this a bit. Here's what I've found:

1. Meteor 1.3.1 does support the es6 importing, but it does not support the class syntax since it's stage-0. They have also disabled custom babel settings through a `.babelrc`, which they plan on turning back on soon. Their current recommendation is to extend the build system.
2. https://github.com/gadicc/meteor-react-hotloader supports custom babel settings, so I'm trying to get that to work, though I haven't had luck so far.

If I can't get the class syntax to work, it's probably not worth moving off Webpack and converting everything to `React.createClass`.

I'll update soon.

**Comment by johnthepink** [comment]
Got the app working with the hot reload package, and added support for classes and decorators. Trying to figure out if I can get it to transpile Apollos as well like webpack was doing. May be worth setting up babel in Apollos and compiling separately. That seems like best practice anyways.

**Comment by jbaxleyiii** [comment]
@johnthepink agreed. Although  hoping we can use typescript  for compilation eventually

**Comment by jbaxleyiii** [comment]
A few things we will need for the move (core and apps):

- [ ] remove all CSS files in favor of other solution
- [ ] either include compilation steps as part of npm scripts or publish to npm (or both)
- [ ] ideally merge in stability branch / typescript

**Comment by johnthepink** [comment]
Are you talking about the CSS modules? Could we use something like this? https://github.com/Quramy/typed-css-modules

**Comment by jbaxleyiii** [comment]
@johnthepink without webpack, I'm not sure how to transform them so the app can use them

**Comment by johnthepink** [comment]
@jbaxleyiii I just slapped this on the app and apollos and it's working like a charm https://github.com/nitive/babel-plugin-css-modules-react

Although, I could totally not understand the issue as well 🐱

**Comment by jbaxleyiii** [comment]
Awesome!

**Comment by johnthepink** [comment]
Fixed a couple more things. Looks like Meteor includes it's own babel runtime, which is causing some issues since it's only up to date with babel 5.

**Comment by johnthepink** [comment]
welp https://github.com/gaearon/babel-plugin-react-transform/issues/57

**Comment by johnthepink** [comment]
I think i'm going to put this down for now. I tried a couple other ideas today. There seems to be issue after issue, and we've got other things we need to add to the app.

I've got everything building correctly on the app side. And i've got a branch of apollos that compiles with babel. Currently, the error I'm getting is related to promises. It's probably an easy fix, but I can't seem to figure it out. If we can get the version I have working, it is WAAAAAY faster than webpack.

```
Uncaught TypeError: Promise is not a constructor
```

It throws when including the reduxSaga middleware. But, when I remove that, it throws on the Lokka package, so I think it's Promises in general that are the problem. 

So, I think the best idea is to pick back up with webpack and start implementing what we need. Maybe wait on Meteor to allow custom babel settings. Or maybe I can pick back up on this soon.
